<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvToolBinarize</class>
 <widget class="QWidget" name="vvToolBinarize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binarize image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="vvToolInputSelectorWidget" name="mToolInputSelectionWidget" native="true"/>
     <widget class="QWidget" name="toolMainWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QRadioButton" name="mRadioButtonGreaterThan">
                  <property name="text">
                   <string>Greater than</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="mRadioButtonEqualThan">
                  <property name="text">
                   <string>Equal</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_3">
                  <property name="text">
                   <string>Not equal</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="vvIntensityValueSlider" name="mThresholdSlider1" native="true"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="mRadioButtonLowerThan">
             <property name="text">
              <string>Lower than</string>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="vvIntensityValueSlider" name="mThresholdSlider2" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Resulting binary image:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="mCheckBoxUseFG">
             <property name="text">
              <string>Use FG</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="vvIntensityValueSlider" name="mFGSlider" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="mCheckBoxUseBG">
             <property name="text">
              <string>Use BG</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="vvIntensityValueSlider" name="mBGSlider" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Interactive (unchek to disable on the fly contours drawing)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vvIntensityValueSlider</class>
   <extends>QWidget</extends>
   <header>vvIntensityValueSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>vvToolInputSelectorWidget</class>
   <extends>QWidget</extends>
   <header location="global">vvToolInputSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mRadioButtonLowerThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>mThresholdSlider2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRadioButtonEqualThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>mRadioButtonLowerThan</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>mRadioButtonLowerThan</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCheckBoxUseBG</sender>
   <signal>toggled(bool)</signal>
   <receiver>mBGSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCheckBoxUseFG</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFGSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
